PPowershell.md

```diff
@@ -0,0 +1,53 @@
+# Plain Powershell | Less is more
+
+![PPowershell](images/PPowershell.png)
+
+## Settings
+```json
+{
+    "fontSize" : 10,
+    "useAcrylic": true,
+    "acrylicOpacity": 0.95,
+    "scrollbarState" : "hidden",
+    "background" : "#272822",
+    "fontFace" : "Cascadia Code PL",
+    "colorScheme": "Campbell Powershell"
+}
+```
+
+## Fonts
+
+- Cascadia Code PL
+
+## Background
+
+You don't need one, very subtle acrylic (.95 opacity) blends the best with any background.
+
+## Color Scheme
+
+```json
+{
+    "name": "Campbell Powershell",
+    "foreground": "#CCCCCC",
+    "background": "#012456",
+    "cursorColor": "#FFFFFF",
+    "black": "#0C0C0C",
+    "red": "#C50F1F",
+    "green": "#13A10E",
+    "yellow": "#C19C00",
+    "blue": "#0037DA",
+    "purple": "#881798",
+    "cyan": "#3A96DD",
+    "white": "#CCCCCC",
+    "brightBlack": "#767676",
+    "brightRed": "#E74856",
+    "brightGreen": "#16C60C",
+    "brightYellow": "#F9F1A5",
+    "brightBlue": "#3B78FF",
+    "brightPurple": "#B4009E",
+    "brightCyan": "#61D6D6",
+    "brightWhite": "#F2F2F2"
+}
+```
+
+### Tip: Use the [oh-my-posh](https://github.com/JanDeDobbeleer/oh-my-posh) with the Agnoster theme to get the best experience.
```

Comment:
Can you please remove this section? Doesn't make much sense in the context because already it's specified in the main README.

![image](https://user-images.githubusercontent.com/1188674/91919418-8a1d3280-ec8b-11ea-98ce-6a67cac2563a.png)

Still, if you think it's not clear, feel free to update the main README accordingly.

Reply:
Hi, Yeah you're right. Putting the tip there doesn't make much since it is already addressed in the main readme.